components/Main/StyledExercises.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const StyledExercises = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  li {
+    margin: 20px;
+    padding: 10px;
+    margin-bottom: 0;
```

Comment:
you can write margin on all 4 sides in one line starting clock-wise from the top: 20px, 20px, 0, 20px (if you only specify 2 values it goes vertically - horizontally)